core/types/index.d.ts

```diff
@@ -6,6 +6,7 @@ declare module '@verdaccio/types' {
   type StorageList = string[];
   type Callback = Function;
   type CallbackError = (err: NodeJS.ErrnoException) => void;
+  type Test = string;
```

Comment:
Is this type necessary? I don’t see usages below

Reply:
You remove when I was writing hahaha

Reply:
I was testing the npm publish :) !